### DISCUSSION TITLE
Em projetos desenvolvidos, quais browsers vocês estão dando suporte?

### DISCUSSION THREAD

**Discussion by theandersonn** [frontendbr/forum#1298] [Discussão]
Como vocês procedem na questão de suporte a browsers?
Notificam o cliente sobre isso? Tipo no briefing.

Algum causo de suporte a browser antigo? Como foi?

**Comment by luizbills** [comment]
Só estou dando suporte ao IE quando solicitado e somente ao IE11. Mas ultimamente o safari está dando mais trabalho.

**Comment by theandersonn** [comment]
@luizbills Qual versão do Safari?

**Comment by luizbills** [comment]
@theandersonn Sei não. Comigo quase sempre o primeiro bug de frontend vem de um iphone usando safari.

**Comment by theandersonn** [comment]
Beleza, obrigado pelo feedback.

**Comment by felipefialho** [comment]
Aqui apenas as versões mais modernas de cada um: Edge, Firefox, Safari, Chrome, etc...

**Comment by theandersonn** [comment]
@LFeh Que legal conseguiram abolir o IE.

**Comment by felipefialho** [comment]
É muito por conta do produto em si também. 

O ideal é verificar ainda enquanto fornece suporte qual a quantidade de acesso e se isso "paga" o custo de produção.
